README.md

```diff
@@ -43,7 +43,7 @@ The terminal client currently has a number of intentional differences to the Zul
   [Hot keys](https://github.com/zulip/zulip-terminal/blob/main/docs/hotkeys.md)
   to better support keyboard-only navigation; other than directional movement
   these also include:
-  - <kbd>z</kbd> - zoom in/out, between streams & topics, or all private messages & specific conversations
+  - <kbd>z</kbd> - zoom in/out, between streams & topics, or all direct messages & specific conversations
```

Comment:
I'd include this either with the documentation-only commit, or *perhaps* with something else that is updating similar text (related to `z`?).

That said, it's fine separate for now - easy to squash with another later.

Reply:
Yeah, I'll include this with the second commit containing the keybindings change.